### DISCUSSION TITLE
Themed/Monochrome icon support

### DISCUSSION THREAD

**Discussion by ghost** [guardianproject/orbot-android#912] [Ideas]
Android 13 has a feature called themed icons which applies the extracted wallpaper colors also to the app icons if a monochrome icon is provided. https://developer.android.com/develop/ui/views/launch/icon_design_adaptive

Let me know your opinion on adding this feature to orbot. I could open a PR which would add support for themed icons. This is the SVG I would use:
![Orbot](https://github.com/guardianproject/orbot/assets/115667885/6bd59fe9-2ae7-45dd-861f-d2ba1acdf8b2)

**Comment by n8fr8** [comment]
Sure, that would be great. We're still settling on the icon for our v17 release, so this will do fine.
